packages/manager/package.json

```diff
@@ -0,0 +1,127 @@
+	"version": "0.1.1-dev-plugins.11",
+	"description": "Manage all aspects of a Slice Machine project.",
+	"repository": {
+		"type": "git",
+		"url": "ssh://git@github.com/prismicio/slice-machine.git"
+	},
+	"license": "Apache-2.0",
+	"author": "Prismic <contact@prismic.io> (https://prismic.io)",
+	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"require": "./dist/index.cjs",
+			"import": "./dist/index.js"
+		},
+		"./client": {
+			"require": "./dist/client.cjs",
+			"import": "./dist/client.js"
+		},
+		"./test": {
+			"require": "./dist/test.cjs",
+			"import": "./dist/test.js"
+		},
+		"./package.json": "./package.json"
+	},
+	"main": "dist/index.cjs",
+	"module": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"typesVersions": {
+		"*": {
+			"*": [
+				"dist/index.d.ts"
+			],
+			"client": [
+				"dist/client/index.d.ts"
+			],
+			"test": [
+				"dist/test/index.d.ts"
+			]
+		}
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"build": "vite build",
+		"dev": "vite build --watch",
+		"format": "prettier --write .",
+		"lint": "eslint --ext .js,.ts .",
+		"prepare": "npm run build",
+		"size": "size-limit",
+		"test": "npm run lint && npm run unit && npm run build && npm run size",
+		"unit": "vitest run --coverage",
+		"unit:watch": "vitest watch",
+		"depcheck": "depcheck --config=.depcheckrc"
+	},
+	"dependencies": {
+		"@prismicio/custom-types-client": "^1.0.2",
+		"@prismicio/types-internal": "^1.5.3",
+		"@slicemachine/plugin-kit": "0.1.8-dev-plugins.11",
+		"@wooorm/starry-night": "^1.4.2",
+		"analytics-node": "^6.2.0",
+		"cookie": "^0.5.0",
+		"cors": "^2.8.5",
+		"file-type": "^18.0.0",
+		"fp-ts": "^2.13.1",
+		"get-port": "^6.1.2",
+		"h3": "^1.0.2",
+		"hast-util-to-string": "^2.0.0",
+		"io-ts": "^2.2.20",
+		"io-ts-reporters": "^2.0.1",
+		"node-fetch": "^3.3.0",
+		"p-limit": "^4.0.0",
+		"prettier": "^2.8.1",
+		"r19": "^0.1.6",
+		"rc9": "^2.0.0",
+		"rehype-stringify": "^9.0.3",
+		"remark-gfm": "^3.0.1",
+		"remark-parse": "^10.0.1",
+		"remark-rehype": "^10.1.0",
+		"semver": "^7.3.8",
+		"unified": "^10.1.2"
+	},
+	"devDependencies": {
+		"@prismicio/mock": "^0.2.0",
+		"@size-limit/preset-small-lib": "^8.1.0",
+		"@types/analytics-node": "^3.1.9",
+		"@types/cookie": "^0.5.1",
+		"@types/semver": "^7.3.13",
+		"@typescript-eslint/eslint-plugin": "~5.46.1",
+		"@typescript-eslint/parser": "^5.46.1",
+		"@vitest/coverage-c8": "^0.25.8",
+		"eslint": "^8.29.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-prettier": "^4.2.1",
+		"eslint-plugin-tsdoc": "^0.2.17",
+		"express": "^4.18.2",
+		"memfs": "^3.4.12",
+		"msw": "^0.49.2",
+		"parse-multipart-data": "^1.5.0",
+		"prettier-plugin-jsdoc": "^0.4.2",
+		"puppeteer": "^19.4.1",
+		"rollup-plugin-rename-node-modules": "^1.3.1",
+		"size-limit": "^8.1.0",
+		"typescript": "^4.9.4",
+		"vite": "^4.0.1",
+		"vite-plugin-sdk": "^0.1.0",
+		"vitest": "^0.25.8"
+	},
+	"peerDependencies": {
+		"puppeteer": "^19"
```

Comment:
`puppeteer` is set as a peer dependency in `@slicemachine/manager` so that it can be downloaded quicker without the need to download and build Chromium.

Puppeteer isn't needed in all use cases, such as running `@slicemachine/init`. We want `@slicemachine/init` to start and run as quickly as possible, so we opted to only install Puppeteer until _after_ `@slicemachine/init` runs. Puppeteer is installed via `slice-machine-ui`'s installation.

`slice-machine-ui` is a "frontend" for `@slicemachine/manager` in the same way `@slicemachine/init` is. Frontends can determine if they need Puppeteer.

Reply:
interesting, it might be worth adding a tech debt ticket to investigate if there's another way to instruct the init to avoid downloading puppeteer,